src/graph.h

```diff
@@ -54,7 +54,20 @@ class Graph {
     [[nodiscard]] std::pair<std::vector<unsigned int>, double>
     getInsertionEdges(std::vector<unsigned int> tour, unsigned int newVertexId) const;
 
+
     double nearestInsertionLoop(unsigned int &start);
+
```

Comment:
These functions from line 59 to 70 shouldn't be here, as they aren't in main and you aren't using them either